### DISCUSSION TITLE
Hiding SQL extension in the URL

### DISCUSSION THREAD

**Discussion by pchemguy** [sqlpage/SQLPage#420] [Ideas]
It would be nice if SQLPage had an option to hide the .sql extension from **displayed** URLs (probably, it should only affect how the URL appears in the browser; another possibility is to permit the use of bare filenames in the code and having SQLPage to use .sql as the default extension for "internal" URLs [in the latter case, the path() function may also need to be adjusted accordingly]).

**Comment by lovasoa** [comment]
This topic has been raised before, the current recommendation is for you to implement that using nginx : https://sql.ophir.dev/your-first-sql-website/nginx.sql

**Reply by pchemguy** [reply]
I see, thank you for clarification.
